iwyu_ast_util.cc

```diff
@@ -1245,6 +1255,14 @@ const NamedDecl* TypeToDeclAsWritten(const Type* type) {
   }
 }
 
+const NamedDecl* TypeToDeclAsWritten(const Type* type) {
+  return TypeToDeclImpl(type, true);
+}
+
+const NamedDecl* TypeToDeclForContent(const Type* type) {
+  return TypeToDeclImpl(type, false);
```

Comment:
Named-arg comment for the Boolean literal.

Reply:
Done.